uint8 Command
uint8 RegisterIndex
uint8 RegisterSelect
uint64 RegisterValues   # low 32 bits: Value0..3
uint8 Counter           # F211_Message_Identifier
